packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 4.2.2 - Feb 7, 2023
+
+Incremental version for Teams Toolkit with multiple bugs fixes and enhancements.
+
+Enhancements:
+
+- Updated pre-requisite checker, applicable app templates and samples to support Node.js 18.
+- Updated comments on Notification bot template for how to send notifications to an individual person.
+- Updated TeamsFx-React SDK to support React 17.
+- Updated TeamsFx SDK, relevant bot app templates and samples to use CloudAdapter.
+- Updated app templates to use TeamsJS SDK v2.7.1.
+
+Bug fixes:
+
+- Fxied an issue where login page will pop up even after signed into Microsoft 365 account from Teams Toolkit.
+- Fixed an issue where it may fail to run ngrok on Windows 11 because of whitespaces in path.
```

Comment:
In my understand the issue is not only for *Windows 11*.
@XiaofuHuang - is that for all *Windows* platforms or all platforms?

Reply:
> In my understand the issue is not only for _Windows 11_. @XiaofuHuang - is that for all _Windows_ platforms or all platforms?

It is for all platform, include mac and linux.

Reply:
Thanks, updated.